### ISSUE TITLE
1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)   out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)

### ISSUE THREAD

**Issue by ritmototal** [comfyanonymous/ComfyUI_examples#31]
Hello,

This might be slowing down my rendering capabilities from what I have been reading a few other people have had this issue recently on fresh installs but I cant seem to find a fix. I have tried uninstalling and reinstalling, but the error persists. Does anyone know how to resolve this? 

C:\Program Files\ComfyUI\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)

**Comment by scaruslooner** [comment]
I have this problem too. Did you find any solutions?
